common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Work.scala

```diff
@@ -151,12 +151,26 @@ sealed trait WorkState {
   def id: String
 }
 
+object InternalWork {
+  // todo: explain why we don't just use Work[Source] because BOOM RECURSION
```

Comment:
Is this still work in progress? Do we want to add a comment now here?

Reply:
How about the comment now?